bindings/C/adios2/c/adios2_c_types.h

```diff
@@ -180,6 +180,13 @@ typedef struct
     adios2_blockinfo *BlocksInfo;
 } adios2_varinfo;
 
+typedef struct
```

Comment:
It is easier to call functions that don't expect or return a `struct`. I would prefer
```C
void adios2_version(int *major, int *minor, int *patch);
const char *adios2_version_str(void);
```
which are overall much simpler to use.